nmea.cc

```diff
@@ -25,6 +25,7 @@
 #include <cstdio>                  // for snprintf, sscanf, fprintf, fputc, stderr
 #include <cstdlib>                 // for atoi, atof, strtod
 #include <cstring>                 // for strncmp, strchr, strlen, strstr, memset, strrchr
+#include <iostream>                // for std::cout
```

Comment:
For consistency I'd prefer to stick with Qt streams and our subclasses.  I acknowledge that the Qt namspace changes introduced in 5.14, e.g. endl -> Qt::endl, are either generating warnings are breaking compatibility with 5.12.

Reply:
The above concern about endl has been taken care of in src/core/logging.h, you can use Qt::endl for all Qt versions we support.